packages/openapi-react-query/README.md

```diff
@@ -25,11 +25,11 @@ Next, generate TypeScript types from your OpenAPI schema using openapi-typescrip
 npx openapi-typescript ./path/to/api/v1.yaml -o ./src/lib/api/v1.d.ts
 ```
 
-> ⚠️ Be sure to <a href="https://redocly.com/docs/cli/commands/lint/" target="_blank" rel="noopener noreferrer">validate your schemas</a>! openapi-typescript will err on invalid schemas.
+> ⚠️ Be sure to <a href="https://redocly.com/docs/cli/commands/lint/" target="_blank" rel="noopener noreferrer">validate your schemas</a>! openapi-typescript will error on invalid schemas.
```

Comment:
“err“ is correct; that is a verb. “Error” is a noun, so this isn’t correct.

However, this sentence only applies to 6.x, not 7.x (since that will validate). So we could just remove this altogether.